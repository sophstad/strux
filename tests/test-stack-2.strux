int main() {
	Stack::string s = new Stack::string();
	s.add("S");
	s.add("H");
	s.add("O");
	s.add("W");
	s.remove();
	s.add("O");
	s.add("W");
	s.remove();
	s.show();
	return 0;
}
