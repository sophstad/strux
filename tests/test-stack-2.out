+------------------------------------------------+ <- Top
| O                                              |
+------------------------------------------------+
| O                                              |
+------------------------------------------------+
| H                                              |
+------------------------------------------------+
| S                                              |
+------------------------------------------------+